utils.js

```diff
@@ -3,5 +3,12 @@ var path = require('path')
 module.exports = {
   isPathRelative: function (p) {
     return path.normalize(p) !== path.resolve(p)
+  },
+  splitCommandLine: function (cmd) {
+    if (cmd) {
+      return cmd.match(/(?:[^\s"']+|["'][^"']*["'])+/g)
```

Comment:
This regex "splits" the `cmd` string on space characters (' ') that are not wrapped in quotes.
For instance, we may have: 
'"google chrome" --incognito'.match(/(?:[^\s"']+|["'][^"']*["'])+/g) ==> ['"google chrome'', '--incognito']

But, yeah, I agree, a comment would be appreciated here... I'll add it ASAP.